cv/lane_detection/src/lane_detection_wrapper_client.py

```diff
@@ -6,51 +6,72 @@
 import numpy as np
 import redis
 import rospy
+import cv2
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
 
 from cv.msg import FloatArray, FloatList
 from geometry_msgs.msg import Point
 from cv_utils import camera_projection
 
 class CVWrapperClient:
     def __init__(self):
-        self.redis = redis.Redis(host='127.0.0.1', port=6379, db=0)
+        self.redis = redis.Redis(host='127.0.0.1', port=6379, db=3)
         self.pub = rospy.Publisher('cv/lane_detections', FloatArray, queue_size=10)
+        self.pub_raw = rospy.Publisher('cv/model_output', Image, queue_size=10)
```

Comment:
Can take this out now ? Although we may wish to have it for the competition for our rosbags (just for debugging)

@1999michael for vis